tests/_kubecli.py

```diff
@@ -0,0 +1,7 @@
+import urllib3
+
+from kubernetes import config
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+config.load_kube_config('./kubeconfig')
```

Comment:
It is nicer (and safer) to place it under a (session) fixture that uses 'autouse'.